packages/binding-wotfirestore/src/codecs/wotfirestore-codec.ts

```diff
@@ -0,0 +1,57 @@
+import * as TD from '@node-wot/td-tools'
```

Comment:
Why do need a decoder for the firestore? the logic seems nothing special just normal text encoding and decoding.

Reply:
@relu91 
Thank you for giving comment.
I am now storing data in Firestore in Buffer format, so encoder/decoder is needed.
I think you are suggesting that since Firestore can store JSON normally, encoder/decoder is unnecessary, and that may be true.
When I was making it, I thought it would be better to design it this way, thinking that it would be able to handle a variety of data types via Buffer.
Was this an unnecessary consideration...

Reply:
If this allows for more data types, then why not. It also shows that we can actually handle such data types in the WoT (and with node-wot). I know that I asked for extra documentation-like stuff a lot but if you can provide code+TD example for such data types, it would be very interesting for node-wot but also for the standardization :)

Reply:
To be honest, currently I haven't come up with an example of such a data type.